light-zk.js/src/transaction/transaction.ts

```diff
@@ -467,6 +469,23 @@ export class Transaction {
           `Root index not found for root${root}`,
         );
       }
+
+      if (merkle_tree_account_data.nextIndex.gte(new BN(255_000))) {
```

Comment:
fyi the tree has 256k leaves, I just chose 255k so that we have a safety margin.

Reply:
Good point. I moved it to a constant with a comment explaining it.